Neos.Flow/Classes/Security/Authentication/Provider/PersistedUsernamePasswordProvider.php

```diff
@@ -12,46 +12,46 @@
  */
 
 use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Configuration\Exception\InvalidConfigurationTypeException;
+use Neos\Flow\ObjectManagement\Exception\CannotBuildObjectException;
+use Neos\Flow\ObjectManagement\Exception\UnknownObjectException;
+use Neos\Flow\ObjectManagement\ObjectManager;
 use Neos\Flow\Security\AccountInterface;
+use Neos\Flow\Security\AccountRepository;
 use Neos\Flow\Security\AccountRepositoryInterface;
 use Neos\Flow\Security\Authentication\Token\UsernamePassword;
 use Neos\Flow\Security\Authentication\Token\UsernamePasswordHttpBasic;
 use Neos\Flow\Security\Authentication\TokenInterface;
 use Neos\Flow\Security\Context;
 use Neos\Flow\Security\Cryptography\HashService;
+use Neos\Flow\Security\Exception as SecurityException;
 use Neos\Flow\Security\Exception\UnsupportedAuthenticationTokenException;
 
 /**
  * An authentication provider that authenticates
  * Neos\Flow\Security\Authentication\Token\UsernamePassword tokens.
  * The accounts are stored in the Content Repository.
  */
-class PersistedUsernamePasswordProvider extends AbstractProvider
+final class PersistedUsernamePasswordProvider extends AbstractProvider
```

Comment:
So, breaking it real hard if we break things, eh? :)

Reply:
Oops, that must have been a reflex => aabb8f6dbed5a56e4f3a49b8f8c38523872edd4e